app/insomnia/enterprise-account-management.md

```diff
@@ -42,19 +42,27 @@ Once the domain is verified, you can configure its settings to control users wit
 
 ### Domain lock
 
-Domain lock allows you to disable access to your Enterprise account for:
-* Existing Hobby users in the domain
-* Uninvited users in the domain
+Use domain lock to remove access to your Insomnia Enterprise account for existing hobby users and uninvited new users.
 
-To enable domain lock, go to your domain settings and click the toggle under **Lock**.
+To enable domain lock:
+- Navigate to your domain settings and click the toggle under **Lock**.
```

Comment:
```suggestion
To enable domain lock, navigate to your domain settings and click the toggle under **Lock**.
```
Specify how to get to domain settings. I couldn't find them. Are they in the app or the account settings?